### DISCUSSION TITLE
post/azure-pipelines-prerequisite

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#64] [Announcements]
# post/azure-pipelines-prerequisite

Mark Heath's Development Blog

https://markheath.net/post/azure-pipelines-prerequisite

**Comment by JMarianczuk** [comment]
I was looking all over for a solution to exactly that problem. The documentation from Microsoft (https://learn.microsoft.com/de-de/windows/uwp/packaging/auto-build-package-uwp-apps) just specifies the YAML without any note of installing the windows sdk. Targetting a different framework version has a slightly different chocolatey command than just changing the version at the end, but a quick search for "windows-sdk" will yield the right command. Thanks for the article :)
